src/js/profile/text.js

```diff
@@ -0,0 +1,20 @@
+import {Component} from "../utils";
+
+export class Text extends Component {
+    constructor(
+        parent,
+        data,
+    ) {
+        super(parent);
+        this.data = data;
```

Comment:
could you please change it to `this_data = data` and then add a getter?

Reply:
Renamed and added a getter